theme_report/README.md

```diff
@@ -1,20 +1,23 @@
 ## Installation instructions
 
-- install [selenium](https://selenium-python.readthedocs.io/getting-started.html):
-  `pip install selenium`
+- install [Playwright for Python](https://playwright.dev/python/docs/intro):
+  Install the Pytest plugin:
+  `pip install pytest-playwright`
+
+  Install the required browsers:
+  `playwright install`
```

Comment:
This is missing a pip install for pandas.

It would probably e easiest to put these all in a requirements.txt and just have one command to install that.

Reply:
Add a requirements.txt file and included all libraries.